app/config/routes.php

```diff
@@ -40,7 +40,13 @@
 
 $app->post('/login', \App\Action\LoginAction::class, 'login');
 
-$app->get('/users', \App\Action\UsersAction::class, 'users');
+$app->get('/users', \App\Action\UserAction::class, 'users');
```

Comment:
Users Actions sounded strange to me. Just found out (http://www.learnersdictionary.com/qa/can-a-plural-noun-act-as-an-adjective) it's a very personal rather than grammatical thing. Also, I took a chance to change something and see how that would impact FW workflow. Can revert if necessary.